src/breadcrumbs/breadcrumbs.scss

```diff
@@ -18,47 +18,67 @@
   user-select: none;
 }
 
-@mixin iui-breadcrumbs-item {
+@mixin iui-breadcrumbs-item($breadcrumbs-height: $iui-component-height) {
+  --_iui-breadcrumbs-item-separator-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' /></svg>");
+
   display: flex;
-  line-height: $iui-component-height;
-  height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
+  align-items: center;
+  line-height: $breadcrumbs-height;
+  height: $breadcrumbs-height;
 
-  a {
-    @include iui-anchor;
+  &:not(:last-child)::after {
+    flex-shrink: 0;
+    width: $iui-icons-small;
+    height: $iui-icons-small;
+    margin: 0 $iui-xxs;
+
+    // #region IE support & future support
+    content: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' fill='%23000' fill-opacity='0.4' /%3E%3C/svg%3E");
+    line-height: 14px;
+
+    @supports (mask: var(--_iui-breadcrumbs-item-separator-svg)) {
+      content: '';
+      background-color: var(--iui-icons-color);
+      mask: var(--_iui-breadcrumbs-item-separator-svg);
+    }
+    // #endregion IE support & future support
+
+    @media (forced-colors: active) {
+      background-color: CanvasText;
+    }
   }
+}
 
-  .iui-button {
-    @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+@mixin iui-breadcrumbs-text {
+  margin: 0 ($iui-s + 1);
+  max-width: 24ch;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  @at-root a#{&} {
+    @include iui-anchor;
```

Comment:
Are we ok with different focus stylings?

![image](https://user-images.githubusercontent.com/81580355/161514556-4e0c2302-0f74-42c7-882f-0d25d8c65b55.png)

Reply:
They are different currently, I don't think it should be an issue.  [I do want to match the width](https://github.com/iTwin/iTwinUI/pull/568#discussion_r842041621) incase we had a scenario where the breadcrumb was switching from hyperlink to button mode, the content won't shift around.

Reply:
Focus styling should be identical now.